python/ray/tests/test_autoscaler.py

```diff
@@ -1029,6 +1029,50 @@ def testScaleUp(self):
         # running_workers metric should be set to 2
         mock_metrics.running_workers.set.assert_called_with(2)
 
+        # Node Updaters have been invoked.
+        self.waitFor(lambda: len(runner.calls) > 0)
+        # The updates failed. Key thing is that the updates completed.
+        self.waitForNodes(
+            2, tag_filters={TAG_RAY_NODE_STATUS: STATUS_UPDATE_FAILED})
+
+    def testScaleUpNoUpdaters(self):
+        """Repeat of testScaleUp with disable_node_updaters=True.
+        Check at the end that no runner calls are made.
+        """
+        config_path = self.write_config(SMALL_CLUSTER)
+        self.provider = MockProvider()
+        runner = MockProcessRunner()
+        mock_metrics = Mock(spec=AutoscalerPrometheusMetrics())
+        autoscaler = StandardAutoscaler(
```

Comment:
Can you parametrize `testScaleUp` with `disable_node_updaters`?

Reply:
Yeah, since the test logic for the two cases differs by like two lines.

Reply:
Seems unittest doesn't work with pytest.mark.parametrize -- instead wrote a helper method that accepts a disable_node_updaters arg and used it in separate tests.